submissions/chris-voitova/html-movie-seat-booking/index.html

```diff
@@ -0,0 +1,318 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Movie Seat Booking</title>
+  </head>
+  <body>
+    <header class="header">
+      <h1 class="title">Choose your movie</h1>
+    </header>
+    <main class="wrapper">
+      <form action="#" class="movie-form">
+        <div class="movie-form__wrapper">
+          <div class="select-wrapper">
+            <label for="movie-select" class="select-label">Select movie</label>
+            <div class="select-block">
+              <select name="movie" id="movie-select" class="select" required>
+                <option class="select__item" value="">Please select</option>
+                <option class="select__item" value="The Matrix">
+                  The Matrix
+                </option>
+                <option class="select__item" value="The Terminator">
+                  The Terminator
+                </option>
+                <option class="select__item" value="Blue Velvet">
+                  Blue Velvet
+                </option>
+              </select>
+            </div>
+          </div>
+          <div class="select-wrapper">
+            <label for="time-select" class="select-label">Select time</label>
+            <div class="select-block">
+              <select name="time" id="time-select" class="select" required>
+                <option class="select__item" value="">Please select</option>
+                <option class="select__item" value="16:00">16:00</option>
+                <option class="select__item" value="18:30">18:30</option>
+                <option class="select__item" value="21:15">21:15</option>
+              </select>
+            </div>
+          </div>
+        </div>
+        <div class="submit-block">
+          <button type="submit" class="button-submit">Continue</button>
```

Comment:
Can you provide me with information about the logic of using this button?

Reply:
This button submits a form for next step to make it easier to work with selects results in js as I thought. Now I see that the logic needs to be rewritten